<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultipleChat Server Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMonitor">
       <attribute name="title">
        <string>运行监控</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_monitor">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_control">
          <item>
           <widget class="QLabel" name="label_port">
            <property name="text">
             <string>监听端口:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPort">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>12345</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStartStop">
            <property name="text">
             <string>启动服务</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_online">
            <property name="text">
             <string>在线人数:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdOnline">
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_log">
          <property name="title">
           <string>系统日志</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_log">
           <item>
            <widget class="QTextBrowser" name="txtLog"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabUsers">
       <attribute name="title">
        <string>用户管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_users">
        <item>
         <widget class="QTableWidget" name="tblUsers">
          <property name="columnCount">
           <number>5</number>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>账号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>昵称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>登录时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>状态</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_user_actions">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnKick">
            <property name="text">
             <string>强制下线 </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
